cookbook/security/voters.rst

```diff
@@ -226,7 +226,7 @@ from the authorization checker is called.
 
             $authChecker = $this->get('security.authorization_checker');
 
-            $this->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
+            $authChecker->denyAccessUnlessGranted('view', $post, 'Unauthorized access!');
```

Comment:
it is a left-over of the time when the controller shortcut was not used in the doc.